contents/docs/new-to-posthog/switch-guide/switching-to-posthog.mdx

```diff
@@ -0,0 +1,118 @@
+/>
+
+***
+
+[Octomind](/customers/octomind):
+
+<Quote
+    imageSource="/images/customers/maria_octomind.jpg"
+    size="sm"
+    name="Maria Zahorcova"
+    title="Chief Marketing Officer, Octomind"
+    quote={`“We’re product-led and it’s really, really important that we keep an eye not only on marketing activities but also the experience beyond that. It was great when I found out I could use PostHog to track data across the site, the docs, and the product — I just had to speak to our engineers about implementing it.”`}
+/>
+
+***
+
+## Advantages for everyone
+
+PostHog is one login and one bill.
+
+That means less headache for more power, with all your data available across a range of products that help you understand user behavior as it happens.
+
+You'll get a responsive team that cares about user needs, delivering improvements and customer requests constantly.
+
+PostHog is completely agnostic to the language, framework and stack you build your product with. You can adopt it incrementally, adding more as needed.
+
+Best of all, PostHog is modern, fast, and built so that you’ll actually enjoy using it.
+
+***
+
+## Persuading product teams
+
+PostHog gives product teams the power to understand users, improve adoption and test assumptions. Your product teammates can get dive in immediately without having to wait for engineers to write new code or hook up an integration.
+
+**[Report anything](/docs/product-analytics/insights)** with charts, trend reporting and [dashboards](/docs/product-analytics/dashboards) that are easy to build, but also deeply customizable for SQL enthusiasts.
+
+**Learn anything you want** about everything the user touches in the product with [autocapture](/docs/product-analytics/autocapture).
+
+**Get clarity about navigation** and user priorities with [heatmaps](/docs/toolbar/heatmaps).
+
+Use [session replay](/docs/session-replay/installation) to **reveal bugs, expose friction, and help instantly answer questions** like “is this new design working?”
+
+**Send events and other details to any other tool**, powering automations and keeping data in sync, with setup you can accomplish in seconds, fully customizable for any API or webhook. [Data pipelines](/docs/cdp) makes it easy.
+
+**Gather feedback** from exactly the users you want to reach with no-code [surveys](/docs/surveys/creating-surveys) that launch instantly.
+
+**Integrate revenue data** from [Stripe](/docs/cdp/sources/stripe) and other sources using data warehouse, so you can get a clear picture of what changes really move the needle.
+
+**Get unblocked by PostHog’s helpful sales and support teams,** ensuring you get the insights and information you need quickly.
+
+***
+
+## Good news for engineering teams
+
+PostHog is designed for developers. We’ve got a dedicated technical staff standing by to help keep you on track. When you reach out to support, you’ll be talking to someone who knows the technical details of the product really well.
```

Comment:
I don't think "support" should be the first point. This one is better at the very least:

> The project is [developed in the open](https://github.com/PostHog/posthog), with a [public roadmap](/roadmap) and [dozens of pull requests landing]([https://github.com/PostHog/posthog/pulls?q=is%3Apr+is%3Aclosed](https://github.com/PostHog/posthog/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aclosed)) each week.